Updated bar file (#1783)
Codecademy/docs: content/matplotlib/concepts/pyplot/terms/bar/bar.md

---

> Title: 'bar'

Reviewer: ```suggestion
Title: '.bar()'
```

> Description: 'A bar is a chart/graph that represents categorical data with rectangular bars.'

Reviewer: ```suggestion
Description: 'Returns a chart/graph that represents categorical data with rectangular bars.'
```

Subjects:
  - 'Data Science'
  - 'Data Visualization'
Tags:
  - 'Graphs'
  - 'Libraries'
  - 'Matplotlib'
CatalogContent:
  - 'learn-python-3'
  - 'paths/computer-science'
---

> A **`bar()`** function returns a chart/graph that represents categorical data with rectangular bars with heights and lengths proportional to the values that they represent. Bars can be plotted vertically or horizontally.

Reviewer: ```suggestion
A **`.bar()`** function returns a chart/graph that represents categorical data with rectangular bars with heights and lengths proportional to the values that they represent. Bars can be plotted vertically or horizontally.
```

## Syntax

> ```plt.bar(x, height, width, bottom, align)```

Reviewer: This should be a ```pseudo code block

Author: @SSwiniarski could you give an example? If the code is written in a descriptive language, ex: plt.bar(x coordinates, height of the bars, width of the bars, and so on).

 I will then remove the parameters. Would this be okay or am I not getting what you mean by pseudo code?

Reviewer: When I say ```pseudo code blocks I am referring to enclosing the content with three backticks followed by the word "pseudo" on its own line, and following with three backtics on its own line. Refer to the [Code](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md#code) and ["Supported Code Blocks" ] (https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md#supported-code-blocks)part of the style guide.

it should look like this:
```` 
```pseudo
plt.bar(x, height, width, bottom, align)
```
````

Author: @SSwiniarski perfect! Thanks for the elaboration.

The parameters to the function are:
  - `x`: The x coordinates of the bars. 
  - `height`: The height(s) of the bars.
  - `width`: The width(s) of the bars default: 0.8  - 
  - `bottom`: The y coordinate(s) of the bars default: 0
  - `align`: {‘center’, ‘edge’}, optional, default: ‘center’

## Example

Goal: Return the value of "X", "Y" und "Z" in a graph representation. The `x` represnts the x-axis while the `y` represnts the y-axis. In the X-axis, we define the "X", "Y" and "Z" parameters that we want to represent. In the y-axis, we define the value of each parameter. The last piece of code specifies the bar color and shows it.

> ```import matplotlib.pyplot as plt

Reviewer: This needs to be a ```py code block

---
Title: '.bar()'

> Description: 'Returns a chart/graph that represents categorical data with rectangular bars.'

Reviewer 2: ```suggestion
Description: 'Returns a chart/graph that represents categorical data with vertical bars.'
```

Subjects:
  - 'Data Science'
  - 'Data Visualization'
Tags:
  - 'Graphs'
  - 'Libraries'
  - 'Matplotlib'
CatalogContent:
  - 'learn-python-3'
  - 'paths/computer-science'
---

> A **`.bar()`** function returns a chart/graph that represents categorical data with rectangular bars with heights and lengths proportional to the values that they represent. Bars can be plotted vertically or horizontally.

Reviewer 2: I think it's typically recommended to use the alternative method .barh() for horizontal charts

## Syntax

``` pseudo
plt.bar(x, height, width, bottom, align)
```

> The parameters to the function are:

Reviewer 2: The language describing these parameters matches or is almost identical to that in that Matplotlib documentation. Please be aware that all content must be original, we do not accept entries that include content copied from other sites.

Author: @caupolicandiaz thanks for your rephrasing the parameters. I wasn't really sure how to rephrase them.

>   - `x`: The x coordinates of the bars. 

Reviewer 2: ```suggestion
  - `x`: Float or array-like values (may be an index and is not required to be scalar). 
```

>   - `height`: The height(s) of the bars.

Reviewer 2: ```suggestion
  - `height`: Float or array-like values designating the heights of the bars.
```

>   - `width`: The width(s) of the bars default: 0.8  - 

Reviewer 2: ```suggestion
  - `width`: Float or array-like values designating the widths of the bars (the default is 0.8). 
```

>   - `bottom`: The y coordinate(s) of the bars default: 0

Reviewer 2: ```suggestion
  - `bottom`: Float or array-like values to designate the y coordinates of the bars (default: 0).
```

>   - `align`: {‘center’, ‘edge’}, optional, default: ‘center’

Reviewer 2: ```suggestion
  - `align`:  Keyword (‘center’ or ‘edge’) designating the bar alignment (default: ‘center’).
```

## Example

> Goal: Return the value of "X", "Y" und "Z" in a graph representation. The `x` represnts the x-axis while the `y` represnts the y-axis. In the X-axis, we define the "X", "Y" and "Z" parameters that we want to represent. In the y-axis, we define the value of each parameter. The last piece of code specifies the bar color and shows it.

Reviewer 2: ```suggestion
Goal: Return the value of "X", "Y" und "Z" in a graph representation. The `x` represnts the x-axis while the `y` represnts the y-axis. In the X-axis, we define the "X", "Y" and "Z" parameters that we want to represent. In the y-axis, we define the value of each parameter. The last piece of code specifies the bar color and displays the graph.
```
